src/main/scala/where/Condition.scala

```diff
@@ -76,3 +77,12 @@ case class Expression(string: String, references: Reference*) extends Condition
   def identifiables: Set[Identifiable] =
     (references flatMap (_.getReferenceable)).toSet
 }
+
+case class NoRelationshipCondition(node: Node) extends Condition {
```

Comment:
also, since you can use this same construction to say "has no relationships of type X", the constructor should allow you to pass in a series of labels to test against (should be empty by default). you could even give it a nice natural constructor like:

```scala
// in the class definition:
def withLabel(label: Label): HasNoRelationships = copy(labels = Seq(label))

// then you get:
node where (HasNoRelationships(node) withLabel "labelName") returns startNode
```